styles/blocks/card.css

```
@@ -0,0 +1,74 @@
+
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  display: block;
+  margin: 0 auto 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin-bottom: 4px;
+}
+
+.card__code-product {
+  color: #616070;
+  margin-bottom: 16px;
+}
+
+.card__mark {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.card__price-text {
+  color: #616070;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card__price-value {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.card__buy-button {
+  display: flex;
+  align-items: center;
+  height: 40px;
+  padding: 11px 69px;
+
+  font-weight: 700;
+  font-size: 14px;
+  color: #fff;
+  text-transform: uppercase;
+
+  border: 1px solid transparent;
+  border-radius: 5px;
+  background-color: #00acdc;
```

for color i think you should use variable,for easy changing your button color.Just create file variables.css and put here
```suggestion
  root:{
  --button-color:#00acdc;
```
and after that
```suggestion
  background-color: var(--button-color);
```